Cargo.toml

```diff
@@ -11,11 +11,11 @@ edition = "2021"
 
 [features]
 default = []
-dev = ["serde_yaml"]
+dev = ["serde_yaml", "walkdir"]
 
 [dependencies]
 atty = "0.2"
-clap = { version = "3.1.18", features = ["derive"] }
+clap = { version = "3.1.18", features = ["derive", "env"] }
```

Comment:
As of clap 3.2.0, this produces numerous spurious warnings that are unrelated to the actual code (I believe, see https://github.com/clap-rs/clap/issues/3822) that may cause CI to fail. It might be worth locking clap to `=3.1.18` for the meantime.